### PR TITLE
Additional testing and validation

### PR DESCRIPTION
**Pull request by payneBrandon**
Adding additional testing to cover empty api key and empty or invalid email values.

### CODE REVIEW

File: src/main/java/jpo/sdw/depositor/DepositorProperties.java [hunk 1]
```diff
@@ -68,20 +70,48 @@ void initialize() {
...
+
+      if(!emailListValid()){
+         logger.error("Email list is not valid email address(es)");
+         throw new IllegalArgumentException("Email list is not valid email address(es)");
+      }
+   }
+
+   private boolean emailValid() {
+      String emailRegex = "^[a-zA-Z0-9_+&*-]+(?:\\." + "[a-zA-Z0-9_+&*-]+)*@" + "(?:[a-zA-Z0-9-]+\\.)+[a-z"
+            + "A-Z]{2,7}$";
```
Comment (trevor-trou) [inline src/main/java/jpo/sdw/depositor/DepositorProperties.java]: I noticed that this expression is different than the expression on line 110. I think the expression on 110 could be simplified to just `^[\w+-.%]+@[\w-]+\.[A-Za-z]{2,4}$` to match a single email. Do we want to make these matchers more consistent?

Comment (payneBrandon) [reply src/main/java/jpo/sdw/depositor/DepositorProperties.java]: The regex on 110 is slightly different and being used to verify a list of emails - we want to be able to send to multiple emails and send from a single one. For consistency, I've updated this regex to be more similar to the list validation (minus the multiple instance capability).

**Review by trevor-trou — APPROVED** [review]
Looks good to me. ~~I do have 1 comment on the RegEx used to validate emails, but that may be outside the scope of this PR.~~
